vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/DefaultExhibition.java

```diff
@@ -0,0 +1,65 @@
+package edu.asu.diging.vspace.core.model.impl;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import edu.asu.diging.vspace.core.model.IDefaultExhibition;
+
+@Entity
+public class DefaultExhibition extends VSpaceElement implements IDefaultExhibition {
+
+	@Id
+	@GeneratedValue(generator = "id-generator")
+	@GenericGenerator(name = "id-generator", 	
```

Comment:
i'm pretty sure you need to give the generator a unique name

Reply:
[VSPC-4] I have given a unique prefix. I'm not sure about unique name for the generator itself. The module and space have the same generator name.